game_eggs/path_of_titans/egg-path-of-titans.json

```diff
@@ -0,0 +1,102 @@
+{
+    "_comment": "DO NOT EDIT IF YOU DO NOT KNOW WHAT YOU ARE DOING!",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2024-04-26T06:39:41+00:00",
+    "name": "Path Of Titans",
+    "author": "danielivanvladimir.revin@gmail.com",
+    "description": null,
+    "features": null,
+    "docker_images": {
+        "ghcr.io\/parkervcp\/yolks:debian": "ghcr.io\/parkervcp\/yolks:debian"
+    },
+    "file_denylist": [],
+    "startup": ".\/AlderonGamesCmd-Linux --game path-of-titans --server true --beta-branch {{BETA_BRANCH}} --auth-token {{AG_AUTH_TOKEN}} --install-dir .\/server\/ && .\/server\/PathOfTitansServer.sh {{SERVER_MAP}}?listen?MaxPlayers={{SERVER_MAX_PLAYERS}} -nullRHI -ServerName={{SERVER_NAME}} -ServerGUID={{SERVER_GUID}} -BranchKey={{BETA_BRANCH}} -Database={{SERVER_DB}} -log",
+    "config": {
+        "files": "{\r\n    \"server.properties\": {\r\n        \"parser\": \"properties\",\r\n        \"find\": {\r\n            \"server-ip\": \"0.0.0.0\",\r\n            \"enable-query\": \"true\",\r\n            \"server-port\": \"{{server.build.default.port}}\",\r\n            \"query.port\": \"7778\"\r\n        }\r\n    }\r\n}",
```

Comment:
Port 7778 is hardcoded and this cant work, when the port is allready assigned in an other server. As i told you, add a variable for this port